oca_dependencies.txt

```diff
@@ -0,0 +1 @@
+server-tools
```

Comment:
Put here
```
server-tools https://github.com/Yajo/server-tools custom-info
```
and you'll be able to test this PR while the other PR is without being merged.